middleware/gzip/gzip_test.go

```diff
@@ -76,24 +76,24 @@ func nextFunc(shouldGzip bool) middleware.Handler {
 	return middleware.HandlerFunc(func(w http.ResponseWriter, r *http.Request) (int, error) {
 		if shouldGzip {
 			if r.Header.Get("Accept-Encoding") != "" {
-				return 0, fmt.Errorf("Accept-Encoding header not expected")
+				return 0, fmt.Errorf("accept-Encoding header not expected")
```

Comment:
Some of these are proper names and should not be lower-cased. Header fields, type names, variable names, etc., should keep their casing. (Content-Encoding, ResponseWriter, etc.)